src/uct/ib/mlx5/ib_mlx5.c

```diff
@@ -370,6 +370,9 @@ static ucs_status_t uct_ib_mlx5_devx_uar_init(uct_ib_mlx5_devx_uar_t *uar,
 {
 #if HAVE_DEVX
     uar->uar            = mlx5dv_devx_alloc_uar(md->super.dev.ibv_context, 0);
+    if (uar->uar == NULL) {
+        return UCS_ERR_NO_MEMORY;
```

Comment:
minor: maybe it is worth to print error/warn here, because this is the first error occurrence in UCX stack

Reply:
I suggest to add some sort error or debug messages. It is royal pain to debug such low level failures.